### PR TITLE
code style improvement 

### PR DESCRIPTION
**Pull request by coswat**
* added `Self` instead of hardcoded impl name -> in notify crate
* organized imports -> in notify crate
* `eprintln!()` macro for Error printing instead of normal println macro -> in file-id crate

### CODE REVIEW

File: file-id/src/lib.rs [hunk 1]
```diff
@@ -85,21 +85,21 @@ pub enum FileId {
 
 impl FileId {
     pub fn new_inode(device_id: u64, inode_number: u64) -> Self {
-        FileId::Inode {
+        Self::Inode {
```
Comment (0xpr03) [inline file-id/src/lib.rs:88]: I think replacing the Name with Self is only reducing readability and doesn't really remove any duplication (FileId -> Self) when you use also `use FileId::*;` and drop the prefix completely. Either way I think it's fine as it is.

File: file-id/bin/file_id.rs [hunk 2]
```diff
@@ -22,6 +22,6 @@ fn print_file_id(path: &str) {
 fn print_result(result: io::Result<FileId>) {
     match result {
         Ok(file_id) => println!("{file_id:?}"),
-        Err(error) => println!("Error: {error}"),
+        Err(error) => eprintln!("Error: {error}"),
```
Comment (0xpr03) [inline file-id/bin/file_id.rs:25]: I mean, you can do that but this is just an example, so it won't matter at all. (Also there is an unrelated import change in that commit.)

**Review by 0xpr03 — CHANGES_REQUESTED** [review]
I would leave the `Self` parts for now, unless you want to use these changes to get towards a bigger feature. Adding only changes for subjective code readability is sadly adding git log/blame noise.

**Comment by coswat** [comment]
oh ok

**Pull request closed without merging**